config.py

```diff
@@ -588,6 +589,18 @@
 
 ica_max_iterations = 200
 
+# ``ica_n_components`` : None | float | int
+#   If int, specifies the number of principal components that are passed to the
+#   ICA algorithm.
+#
+#   If float between 0 and 1, all principal components with cumulative
+#   explained variance less than the value specified here will be passed to
+#   ICA.
+#
+#   If None, all principal components will will be used.
+
+ica_n_components = 60
```

Comment:
ica_n_components should be based on explained variance which will be more robust across datasets

Reply:
maybe 

ica_n_components = 0.99

?

Reply:
maybe 

ica_n_components = 0.99

?

Reply:
Set it to 0.999 again as it was previously.